model.schema.json

```diff
@@ -6,9 +6,61 @@
     "type": "object",
     "properties": {
         "qbicId": {
-            "description": "QBiC internal identifier for sample entities",
+            "description": "QBiC internal identifier for preparation samples",
             "type": "string",
             "pattern": "Q[A-X0-9]{4}[0-9]{3}[A-X0-9]{2}"
+        },
+        "seqFacilityId": {
+            "description": "Name of sample given by the Sequencing Facility",
+            "type": "string"
+        },
+        "sampleName": {
+            "description": "ID of the sample written on the tube",
+            "type": "string"
+        },
+        "sampleGroup": {
+            "description": "Grouping of samples based on shared QBiC internal identifier for entity samples",
+            "type": "string",
+            "pattern": "Q[A-X0-9]{4}ENTITY-[1-9][0-9]*"
+        },
+        "species": {
+            "description": "Scientific name of organism(s) from which the biological material was derived.",
+            "type": "string"
+        },
+        "extractCode": {
+            "description": "QBiC internal identifier for extract samples",
+            "type": "string",
+            "pattern": "Q[A-X0-9]{4}E[1-9][0-9]*"
```

Comment:
This is an experiment code, but should it not be a sample code?

Reply:
Corrected with commit 5a14140